src/print.c

```diff
@@ -53,6 +56,21 @@ int apk_get_screen_width(void)
 	return apk_screen_width;
 }
 
+const char *apk_get_human_size(off_t size, off_t *dest)
+{
+	size_t i;
+	off_t s;
+
+	assert(size >= 0);
+
+	for (i = 0, s = size; s >= 1024 &&
```

Comment:
A late observation. Should it be instead `s >= 10000` like in the existing commit code? Or return and print float with one decimal? I think 1MiB is a bit unprecise/boring output.

Reply:
I have a slight preference towards returning a float, though if `s` is a float and `size` is a `off_t` the assignment `s = size` may not work correctly.

Reply:
An entirely different solution would be making the humanize output optional using a command line flag. If precise size output is desired one can simply disable the humanized output. This might also be useful for scripts.

Reply:
I would just like to merge this sooner than later. I think at this point I'd like to keep the current humanization style like mentioned in https://github.com/alpinelinux/apk-tools/pull/17#discussion_r357023273 and do a follow up patch to add formatting changes as needed. Would this work for you?

Reply:
Just to make sure I understood you correctly: You want me to adjust this PR to match the humanization style currently used in git HEAD. That is, if the size is larger than `10000` bytes then convert it to mebibyte otherwise just display bytes and do not perform any conversions?

Reply:
I mean. Use the `10000` as the boundary when the go up a unit instead of the `1024`.  Thanks.

Reply:
Ah, sorry for the confusion. Done!